### PR TITLE
Traces update k8s instrumentation

### PR DESCRIPTION
**Pull request by mat-rumian**
## Purpose of this pull request

This PR updates guide how to instrument K8s objects using Sumo Logic Kubernetes Collection with OpenTelemetry Operator.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203238071291656

### CODE REVIEW

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/kubernetes.md [hunk 1]
```diff
@@ -6,17 +6,19 @@ sidebar_label: Kubernetes (Beta)
 
 <p> <a href="/docs/beta"><span className="beta">Beta</span></a> </p>
 
-Setting up Tracing instrumentation for Java, Python and NodeJS applications deployed in Kubernetes just got easier. In a few simple steps, with the [OpenTelemetry-Operator](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator) your application is automatically instrumented and your traces are sent to Sumo.
+Setting up Tracing instrumentation for Java, Python, NodeJS and .Net applications deployed in Kubernetes just got easier. In a few simple steps, with the [OpenTelemetry-Operator](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator) your application is automatically instrumented and your traces are sent to Sumo.
```
Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/kubernetes.md:9]:
```suggestion
Setting up Tracing instrumentation for Java, Python, NodeJS, and .NET applications deployed in Kubernetes just got easier. In a few simple steps, with the [OpenTelemetry-Operator](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator) your application is automatically instrumented and your traces are sent to Sumo.
```

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/kubernetes.md [hunk 2]
```diff
@@ -6,17 +6,19 @@ sidebar_label: Kubernetes (Beta)
...
 
 ## Installation
 
-To enable the OpenTelemetry-Operator for the  [Sumo Logic Kubernetes Collection](https://github.com/SumoLogic/sumologic-kubernetes-collection#sumologic-kubernetes-collection), you have to set additional flags during the installation process. The operator will instrument containers belonging to namespaces provided in the `opentelemetry-operator.manager.env.WATCH_NAMESPACE` flag only. The value of the flag is backslash comma separated namespaces list, for example: `opentelemetry-operator.manager.env.WATCH_NAMESPACE="ns1\,ns2\,ns3"`.
+To enable the OpenTelemetry-Operator for the [Sumo Logic Kubernetes Collection](https://github.com/SumoLogic/sumologic-kubernetes-collection#sumologic-kubernetes-collection), you have to set `opentelemetry-operator.enabled=true`.  
+OpenTelemetry Operator needs to know how to instrument containers. For this purpose `Instrumentation` resource has to be created in the namespace where you want to use auto-instrumentation.
```
Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/kubernetes.md:14]:
```suggestion
OpenTelemetry Operator needs to know how to instrument containers. For this purpose, the `Instrumentation` resource must be created in the namespace where you want to use auto-instrumentation. 
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
@mat-rumian , thank you so much for submitting your edits! Made some suggestions here.

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/kubernetes.md [hunk 3]
```diff
@@ -6,17 +6,19 @@ sidebar_label: Kubernetes (Beta)
 
 <p> <a href="/docs/beta"><span className="beta">Beta</span></a> </p>
```
Comment (pbrzoska-sumo) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/kubernetes.md:7]: Should that be removed ?

**Comment by mat-rumian** [comment]
> Do we mention that .net part is in beta ?

@pbrzoska-sumo In the last commit I've added information about .Net instrumentation and mentioned in the note below it's beta.

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/kubernetes.md (hunk 3 above)
Comment (kimsauce) [reply docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/kubernetes.md:7]: Yes, if this is the K8S operator beta->GA

**Review by kimsauce — APPROVED** [review]

**Pull request merged**